MaterialLib/PorousMedium/Permeability/DupuitPermeability.h

```diff
@@ -0,0 +1,49 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <Eigen/Dense>
+
+#include "BaseLib/Error.h"
+#include "MaterialLib/PorousMedium/Permeability/Permeability.h"
+#include "ParameterLib/Parameter.h"
+
+namespace MaterialLib
+{
+namespace PorousMedium
+{
+/// The purpose of the class DupuitPermeability is the implementation of the
+/// special diffusion coefficient \f$h K(h)\f$, where \f$h\f$ is the hydraulic
+/// head and \f$K\f$ is the hydraulic conductivity. The diffusion coefficient is
```

Comment:
`... the hydraulic head and ...` -->`... the hydraulic head of the previous non-linear iteration and ...`